### DISCUSSION TITLE
Feature request: Add support for batch editing

### DISCUSSION THREAD

**Discussion by magico** [excid3/madmin#188] [Ideas]
There are times where I need to edit dozens or even hundreds of items at the same time. For example, updating a specific column to `true` for many instances at a time. It would be amazing if madmin supported this.

**Comment by excid3** [comment]
Definitely would like to do batch actions. 

We can add checkboxes to the rows and a dropdown above of actions. Submitting one of those would send over the IDs or an `all` value of some sort to the matching action.
